third_party/winuwp_h264/H264Encoder/H264Encoder.cc

```diff
@@ -43,6 +43,16 @@ namespace webrtc {
 // QP scaling thresholds.
 static const int kLowH264QpThreshold = 24;
 static const int kHighH264QpThreshold = 37;
+
+// On some encoders (e.g. Hololens) changing rates is slow and will cause
+// visible stuttering, so we don't want to do it too often.
+// todo(fibann): we are ignoring small variations which means the rates might
```

Comment:
The old code did this only with framerate. That alone seems to eliminate most of the stuttering at steady state on HL1 on a local connection. OTOH I don't know what would happen on a worse connection - I would expect WebRTC to make more adjustments to the bitrate as the quality goes up/down, so a threshold on the bitrate changes might smooth the experience. Thoughts?

Reply:
Logged as https://github.com/webrtc-uwp/webrtc-uwp-sdk/issues/208 btw